data/common.yaml

```diff
@@ -20,6 +23,13 @@ mcollective::server_config:
   plugin.yaml: "/etc/puppetlabs/mcollective/generated-facts.yaml"
   registerinterval: 10
   registration: "Choria"
+  logfile: "/var/log/puppetlabs/mcollective.log"
```

Comment:
We could do that, but see above - I think if we keep `$server_config` ect. this might get quite messy as we would need to decide witch setting to keep.

It should be handled in another PR.